examples/llm_plus_gnn/rag_generate_multihop.py

```diff
@@ -0,0 +1,48 @@
+# %%
+import pandas as pd
+import torch
+import tqdm
+from profiling_utils import create_remote_backend_from_triplets
+from rag_feature_store import SentenceTransformerFeatureStore
+from rag_graph_store import NeighborSamplingRAGGraphStore
+
+from torch_geometric.datasets.updated_web_qsp_dataset import (
+    preprocess_triplet,
+    retrieval_via_pcst,
+)
+from torch_geometric.loader import RAGQueryLoader
+from torch_geometric.nn.nlp import SentenceTransformer
+
+# %%
+triplets = torch.load('wikimultihopqa_full_graph.pt')
```

Comment:
this `triplets = torch.load('wikimultihopqa_full_graph.pt')` seems wrong for what we want in an example. you can't expect them to magically have this pt file lying around. needs to download off internet automatically if not locally available already

Reply:
Going to incorporate into a longer Multihop dataset generation example.